kitchen-terraform.gemspec

```diff
@@ -21,7 +32,7 @@ require "rubygems"
   specification.homepage = "https://newcontext-oss.github.io/kitchen-terraform/"
   specification.license = "Apache-2.0"
   specification.add_development_dependency "bundler", "~> 2.0"
-  specification.add_development_dependency "guard-bundler", "~> 2.1"
```

Comment:
What is the motivation for removing the guard-bundler gem? If we do drop it then the Guardfile will need to be updated.

Reply:
This one I can't recall for sure. Maybe I just didn't understand what I was doing! Like I said, I went through the gemspec looking for all requirements I could eliminate. Since the dev dependencies are going with the gem, I thought I would make it lighter weight. I also thought about what are developer preferences vs necessities for CI/CD and distribution. I maybe thought that guard-bundler would be likely to be something a developer could have installed globally if they wanted to use it. Ah, I also thought in terms of omitting the gems.locked for gem distribution. Having bundler install a new gem that appeared in the gemspec would result in the creation of gems.locked. I'll add it back.

Reply:
:thinking: I'm _pretty_ sure that development dependencies are not included when an end user installs a gem via Bundler and rubygems.org, nor are any `gems.lock` included in the repository.